.github/ISSUE_TEMPLATE/ui_bug_report_template.yaml

```diff
@@ -0,0 +1,146 @@
+name: UI Bug Report
+description: Create a report to help us reproduce and correct the UI bug
+labels: ['bug', 'area/uiux']
+title: '[BUG]'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md) for additional information about bug reports. For help with debugging your code, please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
+        #### Please fill in this UI bug report template to ensure a timely and thorough response.
+  - type: dropdown
+    id: contribution
+    attributes:
+      label: Willingness to contribute
+      description: The MLflow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the MLflow code base?
+      options:
+        - Yes. I can contribute a fix for this bug independently.
+        - Yes. I would be willing to contribute a fix for this bug with guidance from the MLflow community.
+        - No. I cannot contribute a bug fix at this time.
+    validations:
+      required: true
+
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: System information
+      description: |
+        Describe the system where you encountered the bug.
+      value: |
+        - **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
```

Comment:
Would this be better as a required checkbox?

Reply:
Good question. To do that, we need to create a separate section. GitHub issue form doesn't support having multiple widgets in one section.

Reply:
btw do we need this question? We normally don't care whether the reproducing code is custom or not.

Reply:
100% agreed. Let's remove it.

Reply:
Agreed!